CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 0.5.0 (September 3, 2024)
+
+FEATURES:
+
+* vault-secrets: CRUD operations for vault-secrets gateway pools [[GH-131](https://github.com/hashicorp/hcp/issues/131)]
+
+IMPROVEMENTS:
+
+* Support loading all variables from Waypoint server in Waypoint agent CLI [[GH-148](https://github.com/hashicorp/hcp/issues/148)]
+* Waypoint Templates: Add support for creating/updating Waypoint Templates with Variables [[GH-130](https://github.com/hashicorp/hcp/issues/130)]
```

Comment:
Should we namespace these as `waypoint: `

Reply:
Oh yes, good point.
Guess we should keep that in mind for PR reviews of the [changlog entry body](https://github.com/hashicorp/hcp/blob/main/.changelog/130.txt#L2) as well. And maybe add some kind of linting of that in the future.